src/components/Shortcuts/ClipboardShortcuts/ClipboardShortcuts.tsx

```diff
@@ -98,21 +103,25 @@ export function ClipboardShortcuts() {
         return;
       }
 
-      const activeElementId = slideInstance.getActiveElementId();
+      const activeElementIds = slideInstance.getActiveElementIds();
 
-      if (!activeElementId) {
+      if (!activeElementIds.length) {
         return;
       }
 
-      const activeElement = slideInstance.getElement(activeElementId);
+      const orderedActiveElementIds =
+        slideInstance.sortElementIds(activeElementIds);
+      const activeElements = slideInstance.getElements(orderedActiveElementIds);
 
-      if (!activeElement) {
+      if (isEmpty(activeElements)) {
         return;
       }
 
-      const content = serializeToClipboard({ elements: [activeElement] });
+      const content = serializeToClipboard({
+        elements: Object.values(activeElements),
+      });
       writeIntoClipboardData(event.clipboardData, content);
-      slideInstance.removeElement(activeElementId);
+      slideInstance.removeElements(activeElementIds);
```

Comment:
This was a last usage of `removeElement` in the code. I think we don't need singular version anymore. Let's delete if from api and use `removeElements` in existing tests.

Reply:
Done